src/v/net/server.cc

```diff
@@ -194,12 +196,22 @@ ss::future<> server::accept(listener& s) {
                   }
               }
 
+              std::optional<security::tls::principal_mapper> tls_pm;
+              auto se_it = std::find_if(
+                cfg.addrs.begin(), cfg.addrs.end(), [&name](const auto& a) {
+                    return a.name == name;
+                });
+              if (se_it != cfg.addrs.end()) {
+                  tls_pm = se_it->principal_mapper;
```

Comment:
if the names match but there is no principal_mapper then I take it that this assignment here leave `tls_pm` in a `nullopt` value state?

Reply:
Correct